pytools/__init__.py

```diff
@@ -468,8 +476,6 @@ def __getattr__(self, name):
 
 
 class Record(RecordWithoutPickling):
-    __slots__: ClassVar[List[str]] = []
```

Comment:
Is there a specific reason this was dropped?

Reply:
I had overlooked that this must be defined in every subclass to be effective 🤦 . Restored in [0a0fcef](https://github.com/inducer/pytools/pull/199/commits/0a0fcef799d5fc19e1e27541b298d524943197a1)

Reply:
Yes, probably. But :shrug: not urgent, we're getting rid of this anyway.